
_____________________________________________________________
### Deep Learning

* _EfficientDet: Scalable and Efficient Object Detection_ ([Link](https://arxiv.org/pdf/1911.09070.pdf))
* _Your Classifier is Secretly an Energy Based Model and You Should Treat it Like One_ ([Link](https://arxiv.org/abs/1912.03263))
* _Gradient Estimation with Stochastic Softmax Tricks_ ([Link](https://arxiv.org/abs/2006.08063))
* _Bootstrap Your Own Latent: A New Approach to Self-Supervised Learning_ ([Link](https://arxiv.org/abs/2006.07733))
* _PatchUp: A Regularization Technique for Convolutional Neural Networks_ ([Link](https://arxiv.org/abs/2006.07794))
* _Domain Extrapolation via Regret Minimization_ ([Link](https://arxiv.org/abs/2006.03908))
* _Denoising Diffusion Probabilistic Models_ ([Link](https://hojonathanho.github.io/diffusion/assets/denoising_diffusion20.pdf))

_____________________________________________________________
### TF Binding

* _Deciphering eukaryotic gene-regulatory logic with 100 million random promoters_ ([Link](https://www.nature.com/articles/s41587-019-0315-8))
* _Intrinsically Disordered Regions Direct Transcription Factor In Vivo Binding Specificity_ ([Link](https://www.cell.com/molecular-cell/fulltext/S1097-2765(20)30352-X?rss=yes))
* _Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome_ ([Link](https://www.biorxiv.org/content/10.1101/168419v4.full.pdf))
* _FactorNet: A deep learning framework for predicting cell type specific transcription factor binding from nucleotide-resolution sequential data_ ([Link](https://www.sciencedirect.com/science/article/pii/S1046202318303293?via%3Dihub))
* _Completing the ENCODE3 compendium yields accurate imputations across a variety of assays and human biosamples_ ([Link](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01978-5#Sec9))
* _Multi-scale deep tensor factorization learns a latentrepresentation of the human epigenome_ ([Link](https://www.biorxiv.org/content/biorxiv/early/2019/04/11/364976.full.pdf))
* _A_ ([Link]())

_____________________________________________________________
### Higher-order interactions

* _Improved Models for Transcription Factor Binding Site Identification Using Nonindependent Interactions_ ([Link](https://www.genetics.org/content/191/3/781.long))
* _A unified approach for quantifying and interpreting DNA shape readout by transcription factors_ ([Link](https://www.embopress.org/doi/full/10.15252/msb.20177902))
* _Learning from mistakes: Accurate prediction of cell type-specific2transcription factor binding_ ([Link](https://www.biorxiv.org/content/biorxiv/early/2018/06/12/230011.full.pdf))
* _Varying levels of complexity in transcription factor binding motifs_ ([Link](https://academic.oup.com/nar/article/43/18/e119/2414334))
* _A General Pairwise Interaction Model Provides an Accurate Description of In Vivo Transcription Factor Binding Sites_ ([Link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0099015))

_____________________________________________________________
### Proteins
* _Improved protein structure prediction using predicted interresidue orientations_ ([Link](https://www.pnas.org/content/117/3/1496
))
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())


_____________________________________________________________
### Attention

* _The utility ofÂ Einsum_ ([Link](https://rockt.github.io/2018/04/30/einsum))
* _The Transformer Family_ ([Link](https://lilianweng.github.io/lil-log/2020/04/07/the-transformer-family.html))
* _Two approaches to faster attention_ ([Link](https://yangkky.github.io/2020/06/22/attention.html))
* _SYNTHESIZER: Rethinking Self-Attention in Transformer Models_ ([Link](https://arxiv.org/pdf/2005.00743.pdf))
* _Linformer: Self-Attention with Linear Complexity_ ([Link](https://arxiv.org/abs/2006.04768))
* _Masked Language Modeling for Proteins via Linearly Scalable Long-Context Transformers_ ([Link](https://arxiv.org/abs/2006.03555))
* _Transforming the language of life: Transformer netowrks for protein prediction tasks_ ([Link](https://www.biorxiv.org/content/10.1101/2020.06.15.153643v1.full.pdf))
* _The proteome landscape of the kingdoms of life_ ([Link](https://www.nature.com/articles/s41586-020-2402-x))


_____________________________________________________________
### Interpretability

* _Explaining Models by Propagating Shapley Values_ ([Link](https://arxiv.org/pdf/1911.11888.pdf))
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())

_____________________________________________________________
### Robustness 

* _Gradient Science Blog -- Madry Lab at MIT_ ([Link](https://gradientscience.org/))
* _Adversarial Training Tutorial_ ([Link](https://adversarial-ml-tutorial.org/))
* _Towards Deep Learning Models Resistant to Adversarial Attacks_ ([Link](https://openreview.net/pdf?id=rJzIBfZAb))
* _Certified Adversarial Robustness via Randomized Smoothing_ ([Link](https://arxiv.org/pdf/1902.02918.pdf))
* _Black-box Smoothing: A Provable Defense for Pretrained Classifiers_ ([Link](https://arxiv.org/pdf/2003.01908.pdf))
* _Adversarial Robustness Through Local Lipschitzness_ ([Link](https://arxiv.org/abs/2003.02460))
* _On the Robustness of Interpretability Methods_ ([Link](https://arxiv.org/pdf/1806.08049.pdf))
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())


_____________________________________________________________
### Normalizing Flows
* _Variational Inference: A Review for Statisticians - Blei et al._ ([Link](https://arxiv.org/pdf/1601.00670.pdf))
* _Variational Inference with Normalizing Flows_ ([Link](https://arxiv.org/pdf/1505.05770.pdf))
* _Why Normalizing Flows Fail to Detect Out-of-Distribution Data_ ([Link](https://arxiv.org/abs/2006.08545))
* _A_ ([Link]())
* _A_ ([Link]())
* _A_ ([Link]())


_____________________________________________________________
### Dimensionality Reduction
* _How to Use t-SNE Effectively_ ([Link](https://distill.pub/2016/misread-tsne/))
* _How Exactly UMAP Works_ ([Link](https://towardsdatascience.com/how-exactly-umap-works-13e3040e1668))
* _Principal Component Analysis Breakdown_ ([Link](https://towardsdatascience.com/principal-component-analysis-breakdown-f3fb1fb48efc))


_____________________________________________________________
### Other
* _Reproducing kernel Hilbert spaces in Machine Learning - Lecture notes_ ([Link](http://www.gatsby.ucl.ac.uk/~gretton/coursefiles/rkhscourse.html))



<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
